ext/ldap/ldap.c

```diff
@@ -3764,9 +3764,7 @@ PHP_FUNCTION(ldap_set_rebind_proc)
 
 	/* callable? */
 	if (!zend_is_callable(callback, 0, NULL)) {
-		zend_string *callback_name = zend_get_callable_name(callback);
-		php_error_docref(NULL, E_WARNING, "Two arguments expected for '%s' to be a valid callback", ZSTR_VAL(callback_name));
-		zend_string_release_ex(callback_name, 0);
+		php_error_docref(NULL, E_WARNING, "Expected argument 2 to be a callable");
```

Comment:
An exception should be thrown instead. You'll find plenty of examples how to do so in https://github.com/php/php-src/pull/5610/files Afterwards, you can add the `callable` type to the parameter in `ldap.stub.php`.

Reply:
Originally, the function accepted ``callable`` or empty string. Empty string had special meaning ("unregister the callback"). I guess, adding ``callable`` will break backward compatibility, as empty string is not a callable. How should we resolve this issue?

Reply:
Done.